src/telepathy/telepathy-stubs.c

```diff
@@ -303,19 +303,6 @@ void sipe_backend_notify_error(SIPE_UNUSED_PARAMETER struct sipe_core_public *si
 			       SIPE_UNUSED_PARAMETER const gchar *title,
 			       SIPE_UNUSED_PARAMETER const gchar *msg) {}
 
-/** USER *********************************************************************/
-
-void sipe_backend_user_feedback_typing(SIPE_UNUSED_PARAMETER struct sipe_core_public *sipe_public,
```

Comment:
I noticed that these symbols were doubly defined when linking so I just assumed that
the code had impl. them, sorry.

Maybe this error has been there all along but just exposed now because sharing pulls in some new code from farstream?